flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/spargel/IterationConfiguration.java

```diff
@@ -189,4 +199,33 @@ public void addBroadcastSetForUpdateFunction(String name, DataSet<?> data) {
 	public List<Tuple2<String, DataSet<?>>> getMessagingBcastVars() {
 		return this.bcVarsMessaging;
 	}
+
+	// ----------------------------------------------------------------------------------
+	// The direction, degrees and the total number of vertices should be optional.
+	// The user can access them by setting the direction, degrees or the numVertices options.
+	// ----------------------------------------------------------------------------------
```

Comment:
all parameters in this class are optional :-)
Can you add javadocs to the new methods, like in the methods above?